titus-ext/aws/build.gradle

```diff
@@ -18,9 +18,12 @@ dependencies {
     compile project(':titus-common')
     compile project(':titus-api')
 
+    compile "io.javaslang:javaslang:2.0.6"
```

Comment:
Set javaslang version in the main gradle file like we do for other dependencies.